pom.xml

```diff
@@ -24,13 +24,14 @@
 
     <properties>
         <jetty.version>9.4.40.v20210413</jetty.version>
-        <kotlin.version>1.3.72</kotlin.version>
-        <kotest.version>4.1.3</kotest.version>
+        <kotlin.version>1.5.20</kotlin.version>
+        <kotest.version>4.6.1</kotest.version>
         <jicoco.version>1.1-89-g9ca691c</jicoco.version>
         <jitsi.utils.version>1.0-93-ge6aba2e</jitsi.utils.version>
         <maven-shade-plugin.version>3.2.2</maven-shade-plugin.version>
         <spotbugs.version>4.1.4</spotbugs.version>
         <jersey.version>2.34</jersey.version>
+        <jackson.version>2.12.4</jackson.version>
```

Comment:
Why is this added?  I don't see it used anywhere?

Reply:
> Why is this added? I don't see it used anywhere?

It is used in other pom.xml files further up.